test/store_test.c

```diff
@@ -81,6 +81,8 @@ static void check_query_key(struct rill_pairs *pairs)
 
 bool test_query_key(void)
 {
+    printf("%s\n", __FUNCTION__);
```

Comment:
FYI, when the time comes I'll remove these as well; just wanted a quick way to see what was going boom.

Reply:
Fixorzed.